test/args/seed-test-args.json

```diff
@@ -1,19 +1,19 @@
 {
   "SeedArguments": {
-    "ADMIN": "0xc039897eE5A0d14A3d1F212922FaF7e159Ab619F",
-    "BENEFICIARY": "0xc039897eE5A0d14A3d1F212922FaF7e159Ab619F",
+    "ADMIN": "0xB86fa0cfEEA21558DF988AD0ae22F92a8EF69AC1",
+    "BENEFICIARY": "0x67DB2ed83cD7F60aa97D84676c10308c4EF14822",
     "TUSDC": "0x8Ab457C4c818153DE31bEF5Aa5D696A7E1EaD267",
-    "TD2D": "0x67DB2ed83cD7F60aa97D84676c10308c4EF14822",
+    "cUSD": "0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1",
     "softCap": "100000000000000000000",
     "hardCap": "10000000000000000000000",
     "price": "1500000000000000000",
-    "startTime": "1660944044",
-    "endTime": "1662148520",
+    "startTime": "5660944044",
+    "endTime": "5662148520",
     "vestingDuration": 31104000,
     "vestingCliff": 1900800,
     "isPermissioned": false,
     "fee": "20000000000000000",
-    "metadata": "0x0000000000000000000000000000000000000000000000000000000000000000"
+    "metadata": "0x516d5a633933537a66587366733254366f5a5344756a614547716742353454416d5870426d485739664169726d62"
```

Comment:
some real data from seed creation via the UI, because `0` string is filtered out, thus gives an error in the FE.